frontend/resourceadm/components/MigrationPanel/MigrationPanel.test.tsx

```diff
@@ -50,6 +50,42 @@ describe('MigrationPanel', () => {
     await waitFor(() => screen.findByText(textMock('resourceadm.migration_not_needed')));
   });
 
+  it('should show message if link service does not exist in given environment', async () => {
+    renderMigrationPanel(
+      {},
+      {
+        getAltinn2DelegationsCount: jest.fn().mockImplementation(() => {
+          return Promise.reject({ response: { status: ServerCodes.NotFound } });
+        }),
+      },
+    );
+    await waitFor(() => screen.findByText(textMock('resourceadm.migration_service_not_found')));
```

Comment:
I believe it is recommended by the creator of react-testing-library to have `expects` in the test, even tho this line kind of works the same way 😄